### ISSUE TITLE
CCXT Pro: ModuleNotFoundError: No module named 'ccxt.async_support.xena'

### ISSUE THREAD

**Issue by pietrodn** [ccxt/ccxt#14880]
I updated to the most recent versions of CCXT Pro and CCXT, but can't import any exchanges because `xena` has been removed in CCXT and CCXT Pro imports it.

- OS: macOS
- Programming Language version: Python 3.10
- CCXT version: ccxt==1.93.4, ccxtpro==1.2.25

```
from ccxtpro import binance
```

```
binance_client.py:12: in <module>
    from ccxtpro import binance
.venv/lib/python3.10/site-packages/ccxtpro/__init__.py:133: in <module>
    from ccxt.async_support.xena import xena                                  # noqa: F401
E   ModuleNotFoundError: No module named 'ccxt.async_support.xena'
```

**Comment by ambrosestartup** [comment]
This also happened to me as well - any ideas guys?

**Comment by GemarPetere** [comment]
I faced ModuleNotFoundError  also when I moved my file into other directory so I decide to back in my desktop directory and it's worked.

**Comment by ambrosestartup** [comment]
Still not working for me, I have my ccxtpro and ccxt packages in my venv folder, and get this error still when importing ccxtpro in main.py next to venv file (see image)

![image](https://user-images.githubusercontent.com/104560551/188643826-257c9793-24bd-4bff-b0b6-b1a24981350d.png)

**Comment by GemarPetere** [comment]
@ambrosestartup is your ccxtpro updated version, kindly consider that also because that is my first solution that worked.

**Comment by ambrosestartup** [comment]
I am using version ccxtpro==1.2.25 and ccxt==1.93.4

**Comment by ambrosestartup** [comment]
I found the fix: Remove line 133 "from ccxt.async_support.xena import xena " from the ccxtpro __init__.py file and it works perfectly now!

**Comment by carlosmiei** [comment]
Hello @pietrodn @GemarPetere @ambrosestartup, thank you all for the report, this is indeed a leftover from Xena delisting but will be fixed very soon

**Comment by kroitor** [comment]
@pietrodn @ambrosestartup thanks for your reports, we've added a fix for that in CCXT Pro 1.2.26+. Pardon for the inconvenience. Your feedback is very much appreciated!

**Comment by pietrodn** [comment]
Thanks!
